hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/classic/ClassicOperationExecutor.java

```diff
@@ -368,15 +365,15 @@ public void execute(Packet packet) {
     @Override
     public void runOnCallingThread(Operation op) {
         if (op == null) {
-            throw new NullPointerException("op can't be null");
+            throw new NullPointerException("Operation can't be null");
```

Comment:
Why did you make this change? The name of the argument should be listed in the exception, not the class. If you want to rename op --> operation, than that is fine to me.